src/lib/CollectStore.js

```diff
@@ -284,8 +284,16 @@ export default class CollectStore {
     // detect oauth case
     const isOAuth = !!account.oauth
 
+    function createDirectoryIfNecessary (folderPath) {
+      if (folderPath) {
+        return cozy.client.files.createDirectoryByPath(folderPath)
+      } else {
+        return Promise.resolve({_id: null})
```

Comment:
Is it not better to explicity put `folder_id` at null in the account info? In this way we say explicitly that this connector doesn't have a folder. This way allows to do that here.